### ISSUE TITLE
Arc theme broken, cannot update nixos unstable

### ISSUE THREAD

**Issue by matthiasbeyer** [NixOS/nixpkgs#29231]
## Issue description

I cannot `nixos-rebuild switch` (or `boot` in my case) as the arc theme is broken.

### Steps to reproduce

`nixos-rebuild` with 

* pkgs.arc-icon-theme
* pkgs.arc-theme

installed.

## Technical details

* System: 17.09pre113138.96457d26dd (Hummingbird)
* Nix version: nix-env (Nix) 1.11.13
* Nixpkgs version: "18.03pre114421.799435b7ca"
* Sandboxing enabled: build-use-sandbox = true

---

```
unpacking source archive /nix/store/5698nycjwg00jb7yvnixn10gb2zanap4-arc-theme-d641d3de1641a9aa0a0f6ac1bacec91d9fdd3326-src
source root is arc-theme-d641d3de1641a9aa0a0f6ac1bacec91d9fdd3326-src
unpacking sources
unpacking source archive /nix/store/wf6hkwipnsn9wnp56f2r1janmk44gj9x-coveralls-1.2.0.tar.gz
patching sources
source root is coveralls-1.2.0
autoreconfPhase
setting SOURCE_DATE_EPOCH to timestamp 1502852780 of file coveralls-1.2.0/setup.cfg
patching sources
configuring
building
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
Download error on https://pypi.python.org/simple/pytest-runner/: [Errno -2] Name or service not known -- Some packages may not be found!
Couldn't find index page for 'pytest-runner' (maybe misspelled?)
Download error on https://pypi.python.org/simple/: [Errno -2] Name or service not known -- Some packages may not be found!
No local packages or working download links found for pytest-runner
Traceback (most recent call last):
  File "nix_run_setup.py", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 49, in <module>
    'Programming Language :: Python :: Implementation :: CPython',
  File "/nix/store/2nsq3cpqy514n5raqdzpfh6lprpqlmjk-python-2.7.13/lib/python2.7/distutils/core.py", line 111, in setup
    _setup_distribution = dist = klass(attrs)
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/setuptools/dist.py", line 320, in __init__
    self.fetch_build_eggs(attrs['setup_requires'])
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/setuptools/dist.py", line 377, in fetch_build_eggs
    replace_conflicting=True,
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/pkg_resources/__init__.py", line 852, in resolve
    dist = best[req.key] = env.best_match(req, ws, installer)
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1124, in best_match
    return self.obtain(req, installer)
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1136, in obtain
    return installer(requirement)
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/setuptools/dist.py", line 445, in fetch_build_egg
    return cmd.easy_install(req)
  File "/nix/store/mssvsm4vni6l73w9zz6188jhixi08rdx-python2.7-bootstrapped-pip-9.0.1/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 667, in easy_install
    raise DistutilsError(msg)
distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pytest-runner')
builder for ‘/nix/store/54jyxv6hkc9i5nkzwyqg1xv1lp530ja7-python2.7-coveralls-python-1.2.0.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/yri0gkj3jlbd3p05252lzwjfkbszmfx0-rtv-1.15.1.drv’: 1 dependencies couldn't be built
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory build-aux
autoreconf: configure.ac: not using Libtool
autoreconf: running: /nix/store/zly5qfsq5fy6axznsza64q83jdsbq711-autoconf-2.69/bin/autoconf --force
autoreconf: configure.ac: not using Autoheader
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:12: installing 'build-aux/install-sh'
configure.ac:12: installing 'build-aux/missing'
autoreconf: Leaving directory `.'
configuring
configure flags: --prefix=/nix/store/fqq0ywkvckvwwlddv297mw94rlm9m516-arc-theme-2016-11-25 --disable-unity --with-gnome=3.24
checking for a BSD-compatible install... /nix/store/bx5zp07kbklb1ichysj5p1kx1qa846lp-coreutils-8.27/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /nix/store/bx5zp07kbklb1ichysj5p1kx1qa846lp-coreutils-8.27/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for pkg-config... /nix/store/278drd32xkvm1w29hj2z2vh28hnfivp1-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
configure: error: Invalid GNOME version: 3.24
builder for ‘/nix/store/1jm729d9gwnh8hxiakm3plmmn7rrln3y-arc-theme-2016-11-25.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/2j91wlbj7kcdx8vmmz0k8l512pvrx742-system-path.drv’: 2 dependencies couldn't be built
```

**Comment by Mic92** [comment]
Mhm weird. Seems to build here: https://hydra.nixos.org/build/60599355 https://hydra.nixos.org/jobset/nixos/release-17.09#tabs-evaluations

**Comment by Mic92** [comment]
Also it seems `coveralls` is broken for you not the theme.

**Comment by Mic92** [comment]
Ok this has been fixed here already: 165fa89762d7156bb3f2774cd3ff0d40cd0a9606
